python_modules/libraries/dagster-looker/dagster_looker/api/resource.py

```diff
@@ -78,6 +82,10 @@ def build_defs(
             request_start_pdt_builds (Optional[Sequence[RequestStartPdtBuild]]): A list of
                 requests to start PDT builds. See https://developers.looker.com/api/explorer/4.0/types/DerivedTable/RequestStartPdtBuild?sdk=py
                 for documentation on all available fields.
+            scheduled_plans_by_dashboard_id (Optional[Dict[str, ScheduledPlanDestination]]): A dictionary
+                which assigns a ScheduledPlanDestination to each dashboard id. This will trigger a scheduled plan
```

Comment:
My first thought was that I'd have to provide a `ScheduledPlanDestination` for each dashboard id, but I don't think it's required based on the implementation below. I'd just avoid using "each" which might be confusing here.

Reply:
Updated to follow the new builder pattern